### PR TITLE
Automatic Angular Synxtax Fixer

### PR DESCRIPTION
**Pull request by wSedlacek**
Adds option to use Angular Syntax `@overide()` when automatically fixing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "tslint-override",
-    "version": "0.6.0",
```
Comment (hmil) [inline package.json]: Don't change the version. I'll take care of it when releasing.

File: package.json [hunk 2]
```diff
@@ -44,9 +44,9 @@
         "@types/shelljs": "^0.8.0",
         "inquirer": "^6.0.0",
         "shelljs": "^0.8.2",
-        "ts-node": "^7.0.0",
```
Comment (hmil) [inline package.json]: Is this necessary for this change?

**Review by hmil — CHANGES_REQUESTED** [review]
Looking pretty good. Few comments:

- The new option should end with `-fixer` to match the syntax of `pascal-case-fixer` since it only affects the behavior of the fixer and not the linter. Try `angular-syntax-fixer`.

File: package.json (hunk 2 above)
Comment (wSedlacek) [reply package.json]: It was just for an `npm audit fix` that's all.

**Comment by wSedlacek** [comment]
Reverted `package.json` changes and updated naming to `angular-syntax-fixer`

File: README.md [hunk 3]
```diff
@@ -165,12 +165,12 @@ Example in VSCode:
 
 ## Contributing, Credits, etc
 
-Author: [@hmil](https://github.com/hmil)
```
Comment (hmil) [inline README.md:168]: Note: The double space at the end of line is important to mark a line break in markdown. I'll re-add it in the next commit

**Review by hmil — APPROVED** [review]

**Comment by hmil** [comment]
Thank you @wSedlacek for another top quality contribution!

**Comment by wSedlacek** [comment]
😅 I'm just happy the automatic fixes will work in my projects.

Thanks for getting it merged in so quickly!

**Pull request merged**